spec/helper.rb

```diff
@@ -0,0 +1,44 @@
+module Helper
+	def omniauth_return
+	hash = {"provider"=>"google_oauth2",
+	 "uid"=>"115821578552534939600",
+	 "info"=>
+	  {"name"=>"Samuel Snider",
+	   "email"=>"samuelsnider.s@gmail.com",
+	   "first_name"=>"Samuel",
+	   "last_name"=>"Snider",
+	   "image"=>"https://lh3.googleusercontent.com/-XdUIqdMkCWA/AAAAAAAAAAI/AAAAAAAAAAA/4252rscbv5M/photo.jpg",
+	   "urls"=>{"Google"=>"https://plus.google.com/115821578552534939600"}},
+	 "credentials"=>
+	  {"token"=>"ya29.GlznBLcfVEQfaAQAEsJ0LNDUlGERaqiV3TtSxw35jw1vpEK0fKLQ0LXuXfQ39vxgaH_FAcfuzIInkMXPtfqyQgcR1MRN78i4288Iuzbn-37y2zXpzinuzfnuRy4EPA",
+	   "expires_at"=>1508278244,
+	   "expires"=>true},
+	 "extra"=>
+	  {"id_token"=>
+	    "eyJhbGciOiJSUzI1NiIsImtpZCI6IjRlZWM5NjBhNmQ1MGE3NGY3OTEzNDZjMmJjN2M3NDZkMjdiZTk2YWYifQ.eyJhenAiOiI3NzUxNzA4Mzg3NC1tZm5zbHRmZWRpNG9naGRmcWp1cmY4ZjVpbnNjaDJsMi5hcHBzLmdvb2dsZXVzZXJjb250ZW50LmNvbSIsImF1ZCI6Ijc3NTE3MDgzODc0LW1mbnNsdGZlZGk0b2doZGZxanVyZjhmNWluc2NoMmwyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE1ODIxNTc4NTUyNTM0OTM5NjAwIiwiZW1haWwiOiJzYW11ZWxzbmlkZXIuc0BnbWFpbC5jb20iLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwiYXRfaGFzaCI6Ii11bDZoVzlNcTVYeXRMUUg0cm5WT1EiLCJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiaWF0IjoxNTA4Mjc0NjQ0LCJleHAiOjE1MDgyNzgyNDR9.f7AxOZ4xyFRF13Wn_ZYLzKS5tnVKfbgzdJHUB8iNrcDXFty40QdrTgnpqOqTVT5-SFdE2Wf4Kv6H_1tV4jc7C2p1yt9zSMCvMLsQ8ToL6DoeCaPdZw1wtnGpX5Zid7hJLp8ndwqTfaB-uNkyceabgmhPGNfLnhAAqccAvJwn9_tn1q4icZSkszFVFdLP0Lr_1KfDSqwaNDjMWilqBC-lMJVTqdQjlcGJd_Zw6K2ZLU06zU5IarmxsMj0mX6RRpwQ-xZh2KSK16NWXKcoyG1OWpfRZf_7KMM7zusoZ_hhTGuxhTAgIYbiS9oRBnU-uT7an8J6GcWbgoyGOxV92irHhA",
```

Comment:
This.

Reply:
The token shouldn't be a problem since they are temporary. The token will expire at a certain point and is no longer valid, so it doesn't compromise security at all by having the token in plaintext on our github.  The user secret or user ID should be treated like password, though, since those allow you to send requests on behalf of a particular client.